pkg/api/v20230701preview/openshiftcluster.go

```diff
@@ -143,6 +181,9 @@ type NetworkProfile struct {
 
 	// The OutboundType used for egress traffic.
 	OutboundType OutboundType `json:"outboundType,omitempty"`
+
+	// The cluster load balancer profile.
+	LoadbalancerProfile *LoadbalancerProfile `json:"loadBalancerProfile,omitempty"`
```

Comment:
Calling out my use of a pointer here, which seems to deviate from the rest of the API structs.  This is a pointer to denote the absence of this struct when outboundType is UserDefinedRouting (no public LB).  I'm open to suggestions if there is a better way.

Reply:
Perfectly fine for this usecase.